.github/workflows/website.yml

```diff
@@ -25,5 +25,4 @@ jobs:
           restore-keys: |
             mkdocs-material-
       - run: pip install mkdocs-material
-      - run: pip install mkdocs-drawio-file
```

Comment:
Incompatible w/ 3.13, but also unused, so just removed. Upvoted the issue upstream so hopefully it gets fixed
If not, there's a drop-in replacement: https://github.com/tuunit/mkdocs-drawio